src/main/resources/config.yml

```diff
@@ -122,6 +122,39 @@ bot:
     roles:
       - "000000000000000000:000000000000000000"
 
+    # A list of options that allow you to notify in a Discord Channel if an account has been unlinked/unlinked
+    notify_linkage:
+
+      # Use separate channels for link/unlink notify
+      # Uses link.channel for both, if false
+      use_separate: false
```

Comment:
Let's just force the user to put the same channel in for both if they want it to be the same